wiki/FitNesseRoot/HsacExamples/TaggedTestcases.wiki

```diff
@@ -0,0 +1,23 @@
+---
+Suite
+---
+!1 Suite To test running a filtered suite from maven
+
+Running this suite using: 
+ * mvn test-compile failsafe:integration-test -DfitnesseSuiteToRun=HsacExamples.TaggedTestcases -DsuiteFilter=runMe -DexcludeSuiteFilter=dontRun
+  * Results in 3 test cases 001, 002 to be executed (all cases that have the tag 'runMe', and don't have the tag 'dontRun')
+ * mvn test-compile failsafe:integration-test -DfitnesseSuiteToRun=HsacExamples.TaggedTestcases -DexcludeSuiteFilter=runMe
+  * Results in 3 test cases 003, 004, 006 to be executed (all cases that don't have the tag 'runMe') 
+ * mvn test-compile failsafe:integration-test -DfitnesseSuiteToRun=HsacExamples.TaggedTestcases -DsuiteFilter=runMe
+  * Results in 3 test cases 001, 002, 005 to be executed (all cases that do have the tag 'runMe')
+  
+Passing a comma separated list to the (exlude)suiteFilter is allowed, so we can also use:
```

Comment:
I agree we should only have 'OR' for exclude filter, so we should also remove the field from the annotation to go to 'and-mode' as the framework does not support it for excludes. Have you seen the pull request I made to FitNesse? In that I also did not have 'AND' support in the exclude.

Let's keep it simple for now and only take on the extra complexity when we find we need 'AND' in our exclude filters...